releasenotes/notes/0.3/0.3-summary-8bdd0273e78a9795.yaml

```diff
@@ -0,0 +1,116 @@
+---
+prelude: >
+    This release has several major changes:
+      - Adds perturbation theory sub module for computing and utilizing the Dyson series
+        and Magnus expansion in a multi-variable setting.
+      - Introduces functionality for simulating Qiskit Pulse schedules using the :class:`.Solver`
+        class.
+      - Adds new solver methods accessible through :func:`.solve_ode` and :func:`.solve_lmde`.
+features:
+  - |
+    Adds support for Python 3.10. Starting with this release
+    Qiskit Dynamics now supports running with Python 3.7 through 3.10.
+  - |
+    The  :meth:`.Solver.solve` method can now run multiple simulations in a single call by
+    specifying one or more of the ``t_span``, ``y0``, and ``signals``
+    arguments as lists of valid single inputs. This supports specifying all as a list of the
+    same length (with a different value for each to be used in each simulation), or a subset
+    can be specified with lists of the same length, with the other arguments giving only a single
+    value (to be used repeatedly while varying the other parameters). Results are returned as a
+    list of the results of the individual simulations. The list of simulations is performed
+    in a serial loop.
+  - |
+    The :class:`.Solver` class has been upgraded to enable Qiskit Pulse schedule simulation.
+    Configuration information for interpeting pulse schedules must be provided at instantiation,
+    and schedules can be solved by passing them via the ``signals`` argument to
+    :meth:`.Solver.solve`.
+  - |
+    Adds :class:`.ArrayPolynomial` for representing multi-variable array-valued polynomials.
+  - |
+    Adds function :func:`.solve_lmde_perturbation` for computing Dyson series and Magnus
+    expansion terms for solutions to linear matrix differential equations.
```

Comment:
I think it would be more descriptive if you combined the solve_lmde_pertrubtation, ArrayPolynomial, and PerturbationResults into a single reno bullet, rather than have them all listed separately.

Reply:
Will combine these.